src/Util/CountryUtility.php

```diff
@@ -18,4 +18,9 @@ public static function isEstonia()
 
         return $countryProvider->getCurrentCountryIsoCode() === Config::ESTONIA_ISO_CODE;
     }
+
+    public static function toLowerCase($string)
+    {
+        return strtolower($string);
```

Comment:
Pretty sure this will throw an error during validation of module.

\Tools::strtolower() should be used.

Reply:
And this should be placed inside StringUtility or something, because of the duplication. 

toLowerCase may be used in other places as well.